cms/__init__.py

```diff
@@ -137,6 +139,20 @@
     LANG_PASCAL: ".o",
 }
 
+COMPILED = 1
+HALF_COMPILED = 3
+INTERPRETED = 5
```

Comment:
This might be way too much of a yolo then; I don't imagine these will be changed before the AIO if they're subtly-off.

We're catering to someone who knows what they're doing; the same algorithm with the same time complexity in different languages should score the same number of marks. The AIO is a special case in the way *we* mark; if submissions timeout because someone didn't realise list.insert() or etc. is O(n) in python *we* treat that differently but the judging software shouldn't.